src/main/java/frc/robot/RobotContainer.java

```diff
@@ -40,7 +43,7 @@
  * subsystems, commands, and trigger mappings) should be declared here.
  */
 public class RobotContainer {
-
+  //TODO: Did I define all the variables correctly? 
```

Comment:
well double check before you make the PR. and no the subsystems should have the final modifier